### PR TITLE
[Workspace] Search in workspace

### PR DESCRIPTION
**Pull request by Gustav-Eikaas**
# Description

Please include a summary of the change and which issue is fixed.
List any external dependencies that are required for this change.

Completes: AB#FILL_IN_YOUR_ISSUE_ID

## Checklist:

-   [ ] I have performed a self-review of my own code.
-   [ ] I have linked my DevOps task using the AB# tag.
-   [ ] My code is easy to read, and comments are added where needed.
-   [ ] My code is covered by tests.

### CODE REVIEW

File: src/Core/WorkSpace/src/Components/DataViewerHeader/Header.tsx [hunk 1]
```diff
@@ -104,12 +105,13 @@ export const CompletionViewHeader = ({
...
                         <>
                             <TabButton
+                                width={'48px'}
                                 onClick={factory.onClick}
                                 aria-selected={false}
                                 title={factory.title}
                             >
                                 <Icon name={'add'} />
-                                {factory.title}
+                                {/* {factory.title} */}
```
Comment (Gustav-Eikaas) [inline src/Core/WorkSpace/src/Components/DataViewerHeader/Header.tsx:114]: Uncomment

File: src/Core/WorkSpace/src/Components/DataViewerHeader/Header.tsx [hunk 2]
```diff
@@ -145,7 +147,7 @@ export const CompletionViewHeader = ({
                         })}
                     </>
                     <Divider />
-                    {/* <SearchButton /> */}
+                    <SearchButton />
```
Comment (Gustav-Eikaas) [inline src/Core/WorkSpace/src/Components/DataViewerHeader/Header.tsx:150]: Comment out

File: src/Core/WorkSpace/src/Components/ToggleButton.tsx [hunk 3]
```diff
@@ -13,16 +14,17 @@ const Button = styled.button<BoxProps>`
     position: relative;
     white-space: nowrap;
     text-overflow: ellipsis;
-    overflow-x: hidden;
+    /* overflow-x: hidden; */
```
Comment (Gustav-Eikaas) [inline src/Core/WorkSpace/src/Components/ToggleButton.tsx:17]: remove

File: src/Core/WorkSpace/src/Components/DataViewerHeader/HeaderStyles.ts [hunk 4]
```diff
@@ -17,11 +17,13 @@ export const TitleBar = styled.div`
 `;
 
 export const ActionBar = styled.div`
-    display: flex;
+    /* display: flex; */
     height: 48px;
-    flex-direction: row;
-    justify-content: space-between;
```
Comment (Gustav-Eikaas) [inline src/Core/WorkSpace/src/Components/DataViewerHeader/HeaderStyles.ts:23]: remove comments

**Review by jaysencpp — APPROVED** [review]

**Pull request merged**